boards/px4/fmu-v5/init/rc.board_sensors

```diff
@@ -22,6 +22,13 @@ hmc5883 -C -T -X start
 qmc5883 -X start
 lis3mdl -X start
 
+# ICM20948 as external magnetometer on I2C (e.g. Here GPS)
+if ! icm20948 -X -M -R 6 start
+then
+  # external emulated AK09916 (Here2) is rotated 270 degrees yaw
+  ak09916 -X -R 6 start
```

Comment:
Is the AK09916 rotated 270 degrees in your particular setup or within the actual Here2?

Reply:
Seems like inside the Here2, I just copied all this from the fmu-v3 init

Reply:
That's kind of horrible depending on if that's specific to the Here2 or the driver implementation. We'll likely have to review this later (especially if someone else starts using an AK09916).